docker/test/fasttest/run.sh

```diff
@@ -194,7 +194,12 @@ function build
 {
     (
         cd "$FASTTEST_BUILD"
-        time ninja clickhouse-bundle 2>&1 | ts '%Y-%m-%d %H:%M:%S' | tee "$FASTTEST_OUTPUT/build_log.txt"
+        TIMEFORMAT=$'\nreal\t%3R\nuser\t%3U\nsys\t%3S'
```

Comment:
Is `TIMEFORMAT` only used by `time`?

Reply:
Yes, to print `75.431` instead of `1m15.431s`